src/components/App.js

```diff
@@ -0,0 +1,20 @@
+import PropTypes from 'prop-types';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+const App = ({ result }) => (
```

Comment:
- [ ] App component shouldn't receive any props, please remove the result props passed to App.
- [ ] You can also leave the Display component without passing result props to it for now.